src/app.ts

```diff
@@ -1,11 +1,13 @@
 import express, { Application } from 'express';
 import cors from 'cors';
 import bodyParser from 'body-parser';
-import { commonErrorHandler } from './lib/common-error-handler';
 
+import { commonErrorHandler } from './lib/common-error-handler';
 import { connectDb } from './db/db';
 import { createDataSourceRouter } from './data-source.router';
 import { MessageBroker } from './rabbitmq/rabbitmq';
+import config from 'config';
+import Keycloak from 'keycloak-connect';
```

Comment:
These are external dependencies. Place them right after the body-parser.

Reply:
fixed